backend/user/urls.py

```diff
@@ -2,7 +2,20 @@
 from .views import *
 from django.conf import settings
 from django.conf.urls.static import static
+from rest_framework import permissions
+from rest_framework_swagger.views import get_swagger_view
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Django Sample Application API",
+        default_version='v1',
+        description="Welcome to the Django Sample Application API documentation",
```

Comment:
Let's change this to be like 'SWE574 G1 Memories Application API Documentation'.

Reply:
I changed the Title! 🦤